tests/filter_exclude/test

```diff
@@ -69,55 +69,55 @@ $subj_clr = $subj_sen unless defined $subj_clr;
 # try adding rule for each supported field type and test for (a few)
 # unsupported types
 $result = system("auditctl -a exclude,always -F msgtype=$msgtype");
-ok( $result, 0 );
+ok( $result, 0 );    # add msgtype ok?
```

Comment:
Please don't do this; the operation which is being checked is the line **directly above this one**.  This is comment noise and it detracts from other comments in the test suite.

This applies to all the other similar comments you've added in this PR; please remove them and leave the code as-is.